server/backend/warehouse/window_test.go

```
@@ -0,0 +1,64 @@
+	}
+	today := day("2026-08-31")
+
+	cases := []struct {
+		name       string
+		from, to   string
+		histEmpty  bool
+		freshEmpty bool
+		hEnd       string
+		fStart     string
+	}{
+		{"entirely past", "2026-08-01", "2026-08-31", false, true, "2026-08-31", ""},
```

golangci-lint (2.12.2)

[error] string `2026-08-31` has 8 occurrences, make it a constant

(goconst)